desktop_gui.py

```diff
@@ -26,19 +24,20 @@ def load_config():
     return config
 
 config = load_config()
-print(" memory length is ", config['memory_length'])
-print(" summary update interval is ", config['summary_update_interval'])
+chat_history_enabled = config['chat_history']
+branching_enabled = config['branching']
+conversations_enabled = config['conversations']
```

Comment:
For cases like this where we are using the same blocks of code/making the same changes, we can define an initialization function we can call at the top/main section of each script to streamline things and for quality of life